libraries/AP_Parachute/AP_Parachute.h

```diff
@@ -20,6 +20,10 @@
 
 #define AP_PARACHUTE_ALT_MIN_DEFAULT            10     // default min altitude the vehicle should have before parachute is released
 
+#define AP_PARACHUTE_EMERGENCY_DEFAULT          0       // default value to emergency parachute
+#define AP_PARACHUTE_MAX_SINK_DEFAULT           10.0    // default maximun sink speed in m/s to trigger emergency parachute
+#define AP_PARACHUTE_MIN_BATT_VOLTAGE_DEFAULT   10.0    // default minimum battery voltage to trigger emergency parachute
```

Comment:
This is unneeded now that you are leveraging the failsafe system.

Reply:
We should keep emergency and max sink default values, i will remove battery for now.